src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -1,20 +1,19 @@
 package lotto
 
 class LottoMachine() {
-    fun createTickets(userAmount: Int): List<Lotto> {
+    fun createGeneratedTickets(
+        totalPurchasedTickets: Int,
+        manualTicketsQuantity: Int,
+    ): List<Lotto> {
         val lottos = mutableListOf<Lotto>()
-        val amountOfTickets = calculateTickets(userAmount)
-        var count = 0
-        while (count < amountOfTickets) {
+        var ticketsToBeGenerated = totalPurchasedTickets - manualTicketsQuantity
```

Comment:
```suggestion
        val ticketsToBeGenerated = totalPurchasedTickets - manualTicketsQuantity
```

It'll be great to keep things `val` when possible. :)